tests/unittests/MLTest.cpp

```diff
@@ -955,11 +983,16 @@ TEST_P(MLTest, testFindPixelRegression) {
 
 INSTANTIATE_TEST_CASE_P(Interpreter, MLTest,
                         ::testing::Values(BackendKind::Interpreter));
-
 #ifdef GLOW_WITH_CPU
 INSTANTIATE_TEST_CASE_P(JIT, MLTest, ::testing::Values(BackendKind::CPU));
 #endif
-
 #ifdef GLOW_WITH_OPENCL
 INSTANTIATE_TEST_CASE_P(OpenCL, MLTest, ::testing::Values(BackendKind::OpenCL));
 #endif
+
+INSTANTIATE_TEST_CASE_P(Interpreter, InterpreterAndCPU,
+                        ::testing::Values(BackendKind::Interpreter));
+#ifdef GLOW_WITH_CPU
+INSTANTIATE_TEST_CASE_P(InterpreterAndCPU, MLTest,
```

Comment:
we already instantiate `MLTest` with CPU, what is this for?

Reply:
I want to disable OpenCL that does not support quantization.

Reply:
Good catch!